build/download-saturn-l2.js

```diff
@@ -31,7 +33,7 @@ async function main () {
   await Promise.all(
     assets
       .map(async ({ name, browser_download_url: url }) => {
-        const match = name.match(/^L2-node_\d+\.\d+\.\d+_([A-Za-z0-9]+)_([A-Za-z0-9_]+)\.tar\.gz$/)
+        const match = name.match(/^L2-node_([A-Za-z0-9]+)_([A-Za-z0-9_]+)\.(tar\.gz|zip)$/)
```

Comment:
the version number was removed from the file name, plus apple requires `.zip` for notarization

Reply:
> the version number was removed from the file name,

Hmm, that's not great. I believe it's a best practice to include the version number in the file names of release artefacts.

Is it difficult to fix the L2-node release process to bring back the version number to file names?

I am ok to land what your have now and change this script again later, if/when we add the version number back.

Reply:
The issue is that `gon` doesn't support variables in all fields, so we can't script `zip.output_path`. We could however create a script that renames the artefacts after the fact.

If you look at gon itself for example, they don't have the version number there: https://github.com/mitchellh/gon/releases/tag/v0.2.5.
Goreleaser seems to have assets both with and without version number: https://github.com/goreleaser/goreleaser/releases/tag/v1.11.5, but there's more without so I assume also for them this is the main way to go?

> I believe it's a best practice to include the version number in the file names of release artefacts
I don't think I've heard of that, do you have a source?

Reply:
> The issue is that `gon` doesn't support variables in all fields, so we can't script `zip.output_path`. We could however create a script that renames the artefacts after the fact.

I see. I would consider this topic as "nice to have", nothing major.

> > I believe it's a best practice to include the version number in the file names of release artefacts
> 
> I don't think I've heard of that, do you have a source?

Ha, you got me! Maybe it's just my assumption based on the filenames I have seen on various download pages.

Few examples:

- Node.js: https://nodejs.org/dist/v16.17.1/node-v16.17.1-darwin-x64.tar.gz
- An example Rust project: https://github.com/dandavison/delta/releases/download/0.14.0/delta-0.14.0-x86_64-apple-darwin.tar.gz
- PHP: https://www.php.net/distributions/php-8.1.11.tar.gz
- Git for Windows: https://github.com/git-for-windows/git/releases/download/v2.38.0.windows.1/Git-2.38.0-64-bit.exe